src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -721,6 +721,7 @@ internal bool IsEmpty
             /// Add new item to the tail of the queue.
             /// </summary>
             /// <param name="item">The item to add.</param>
+            /// <param name="emptyToNonEmptyListTransitionCount"></param>
```

Comment:
💡 Consider disabling CS1573 (missing parameter doc) and CS1712 (missing type parameter doc)

Reply:
> Consider disabling CS1573 (missing parameter doc) and CS1712 (missing type parameter doc)

I have.  I just noticed this one and added it figuring at some point we'll enable those.  These don't actually break source.dot.net... just the malformed XML ones.